src/sentry/integrations/example/integration.py

```diff
@@ -65,14 +66,21 @@ def dispatch(self, request: Request, pipeline) -> HttpResponse:
 )
 
 
-class ExampleIntegration(IntegrationInstallation, IssueSyncMixin, RepositoryMixin):
```

Comment:
Noob question, how is Example used?

Reply:
it's used in tests, e.g. https://github.com/getsentry/sentry/blob/4642ebb073341ee976c3cf2f4550c93865749691/tests/sentry/models/test_groupassignee.py#L119-L135

Reply:
Cool 🫘 
Do you plan to eventually complete its methods to have more complete tests then?

Reply:
i'm actually not sure what the benefit of having this is versus just using an existing integration, i would need to dig more

Reply:
For dummy implementations that we expect not to get called, it might be better to raise `NotImplementedError`, maybe with a message like "assumed not to be needed for test cases".

It would also be good to move this to `testutils` to help communicate its purpose (but that doesn't need to be part of this PR).